P5/Source/Specs/post.xml

```diff
@@ -0,0 +1,252 @@
+            <textNode/>
+            <classRef key="model.headLike"/>
+            <elementRef key="opener"/>
+            <classRef key="model.pLike"/>
+            <classRef key="model.gLike"/>
+            <classRef key="model.phrase"/>
+            <classRef key="model.inter"/>
+            <classRef key="model.global"/>
+            <elementRef key="lg"/>
+            <classRef key="model.lLike"/>
+            <classRef key="model.divBottom"/>
+        </alternate>
+    </content>
+    <attList>
+        <attDef ident="modality" usage="rec" mode="add">
+            <desc xml:lang="en" versionDate="2021-09-07">written or spoken mode</desc>
+            <datatype minOccurs="1" maxOccurs="1">
+                <dataRef key="teidata.enumerated"/>
+            </datatype>
+            <valList xml:lang="en" type="semi">
+                <valItem ident="written">
+                    <desc xml:lang="en" versionDate="2021-09-07">for stretches of text and/or multimodal elements</desc>
+                </valItem>
+                <valItem ident="spoken">
+                    <gloss xml:lang="en" versionDate="2021-09-07">for audio posts</gloss>
+                </valItem>
+            </valList>
+            <remarks xml:lang="en" versionDate="2021-09-07">
+                <p>This attribute is intended to convey information about the CMC environment in
+                    which the post was created, as opposed to the MIME type of the content conveyed,
+                    which is indicated by the elements contained within the <gi>post</gi>.</p>
+            </remarks>
+        </attDef>
+        <attDef ident="replyTo" mode="add">
+            <desc xml:lang="en" versionDate="2021-09-07">indicates to which previous post the
+                current post replies or refers to. This attribute should be used to encode
+                "technical" reply information, i.e. which is due to a formal reply action (such as
+                activating a "reply" button in the client software) and which is also formally
+                represented in the source, e.g. in the "references" field of a Usenet message header
+                or in the subject line of a forum post. It should rather not be used for inferred,
+                or interpreted reply relations such as based on linguistic discourse markers, nor
+                for the indentation relations in Wiki talk pages.
+                <!--See Lüngen/Herzberg (2019)--></desc>
+            <datatype minOccurs="1" maxOccurs="unbounded">
+                <dataRef key="teidata.pointer"/>
+            </datatype>
+        </attDef>
+        <attDef ident="indentLevel" mode="add">
+            <desc xml:lang="en" versionDate="2021-09-07">marks the level of indentation of the
+                current post in a thread-like structure (as defined by its author and in relation to
+                the standard level of indentation which should be encoded as indentLevel="0").
```

Comment:
Missing inline elements. A suggestion: “which should be encoded as `<val>0</val>`”

Reply:
Has been moved to att.indentation.xml, and has been encoded (as `<val>`) there.